gravitee-apim-console-webui/src/portal/customization/top-bar/portal-top-bar.component.html

```diff
@@ -0,0 +1,116 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<portal-header title="Top Bar" subtitle="Customize the Developer Portal top bar." />
+
+<mat-card class="portal-top-bar">
+  <mat-card-content>
+    <div class="portal-top-bar__header">
+      <div class="portal-top-bar__header__title">
+        <h3 class="portal-top-bar__header__title__main">Menu links</h3>
+        <div class="portal-top-bar__header__title__subtitle">Customize the Developer Portal top bar menu with up to 5 links.</div>
+      </div>
+
+      <div class="portal-top-bar__header__action">
+        <button
+          [disabled]="!canAddLink"
+          (click)="onAddLinkClick()"
+          mat-raised-button
+          color="primary"
+          aria-label="add-link"
+          data-testid="add_link_button"
+        >
+          <mat-icon>add</mat-icon> Add Link
+        </button>
+      </div>
+    </div>
+    <table
```

Comment:
If a user is in read-only, perhaps we can show the link address in the table? That way they can at least see what the target is?

Reply:
Not yet